docs/conceptual/runtime/frame.md

```diff
@@ -93,191 +93,191 @@ authorities.
 phragmen](https://research.web3.foundation/en/latest/polkadot/NPoS/4.%20Sequential%20Phragm%C3%A9n%E2%80%99s%20method/).
 
 * [Docs](https://substrate.dev/rustdocs/master/pallet_elections_phragmen/index.html)
 * [Source](https://github.com/paritytech/substrate/blob/master/frame/elections-phragmen/src/lib.rs)
 
 ### Elections
 
-The Elections module is an election module for stake-weighted membership selection of a collective.
+The Elections pallet is an election module for stake-weighted membership selection of a collective.
 
 * [Docs](https://substrate.dev/rustdocs/master/pallet_elections/index.html)
 * [Source](https://github.com/paritytech/substrate/blob/master/frame/elections/src/lib.rs)
 
 ### EVM
 
-The EVM Module is an [Ethereum](https://en.wikipedia.org/wiki/Ethereum) virtual machine (EVM)
+The EVM pallet is an [Ethereum](https://en.wikipedia.org/wiki/Ethereum) virtual machine (EVM)
 execution module for Substrate.
 
 * [Docs](https://substrate.dev/rustdocs/master/pallet_evm/index.html)
 * [Source](https://github.com/paritytech/substrate/blob/master/frame/evm/src/lib.rs)
 
 ### Example
 
-The Example module is a simple example of a runtime module demonstrating concepts, APIs and
-structures common to most runtime modules.
+The Example pallet is a simple example of a pallet demonstrating concepts, APIs and
+structures common to most pallets.
 
 * [Docs](https://substrate.dev/rustdocs/master/pallet_example/index.html)
 * [Source](https://github.com/paritytech/substrate/blob/master/frame/example/src/lib.rs)
 
 ### Finality Tracker
 
-The Finality Tracker module tracks the last finalized block, as perceived by block authors.
+The Finality Tracker pallet tracks the last finalized block, as perceived by block authors.
 
 * [Docs](https://substrate.dev/rustdocs/master/pallet_finality_tracker/index.html)
 * [Source](https://github.com/paritytech/substrate/blob/master/frame/finality-tracker/src/lib.rs)
 
 ### Generic Asset
 
-The Generic Asset module provides functionality for handling accounts and asset balances.
+The Generic Asset pallet provides functionality for handling accounts and asset balances.
 
 * [Docs](https://substrate.dev/rustdocs/master/pallet_generic_asset/index.html)
 * [Source](https://github.com/paritytech/substrate/blob/master/frame/generic-asset/src/lib.rs)
 
 ### GRANDPA
 
-The GRANDPA module extends GRANDPA consensus by managing the GRANDPA authority set ready for the
+The GRANDPA pallet extends GRANDPA consensus by managing the GRANDPA authority set ready for the
 native code.
 
 * [Docs](https://substrate.dev/rustdocs/master/pallet_grandpa/index.html)
 * [Source](https://github.com/paritytech/substrate/blob/master/frame/grandpa/src/lib.rs)
 
 ### I'm Online
 
-The I'm Online module allows validators to gossip a heartbeat transaction with each new session to
+The I'm Online pallet allows validators to gossip a heartbeat transaction with each new session to
 signal that the node is online in the current era.
```

Comment:
```suggestion
signal that the node is online.
```

Reply:
Just for my own understanding, @joepetrowski are you suggesting to remove "in the current era" because it is incorrect, or just because we haven't and shouldn't introduce the concept of eras here?

Reply:
Sort of both. It is once per _session,_ not era.